### ISSUE TITLE
dbt-trino adds comment into table create statement by default

### ISSUE THREAD

**Issue by alaturqua** [starburstdata/dbt-trino#343]
### Expected behavior

dbt-core has persist_docs by default disabled. And dbt-trino should behave same as dbt-core.

https://docs.getdbt.com/reference/resource-configs/persist_docs

Additionally documentation is missing on official dbt documentation page. Trino is not listed on supported adapters list.

- https://docs.getdbt.com/reference/resource-configs/persist_docs
- https://docs.getdbt.com/reference/resource-configs/trino-configs

### Actual behavior

dbt-trino adds comment on table creation. When a description for a model exists on schema.yml

``` 
create table iceberg.<schema>.<table>
    comment ''
    as (
    SELECT
     *
    FROM
    table
    )
 ```

### Steps To Reproduce

- Do not set persist_docs config anywhere.
- Run a model with dbt trino where for a model description exists.

### Operating System

Windows 10

### dbt version

dbt-core==1.6.0, dbt-trino==1.6.1

### Trino Server version

422

### Python version

3.10

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by alaturqua** [comment]
Thanks for fixing this. Is there any estimation, when there will be the next release including this fix?

**Comment by damian3031** [comment]
@alaturqua It has just been released in 1.6.2!
